projects/core/src/occ/config/occ-config-validator.ts

```diff
@@ -8,4 +8,7 @@ export function occConfigValidator(config: OccConfig) {
   ) {
     return 'Please configure backend.occ.baseUrl before using storefront library!';
   }
+  if (!config.backend?.occ?.prefix?.startsWith('/')) {
```

Comment:
Validator should never change the configuration. I would move it to private methods in `OccSitesConfigLoader` and `OccEndpointsService`. Maybe later `OccEndpointsService` should use `OccEndpointsService` for utilities? Just a thought